i18n/en.json

```diff
@@ -4112,11 +3588,11 @@
   },
   {
     "id": "model.client.set_team_icon.no_file.app_error",
-    "translation": ""
+    "translation": "No file under 'image' in request"
   },
   {
     "id": "model.client.set_team_icon.writer.app_error",
-    "translation": ""
+    "translation": "Unable to write request"
```

Comment:
Which request?

Reply:
A similar block of code than the previous one. In this case, it fails when it tries to close the image file (Understand close as finalize the upload after all the write needed is already done).